datacapture/src/test/java/com/google/android/fhir/datacapture/utilities/MoreLocalDatesTest.kt

```diff
@@ -49,4 +54,109 @@ class MoreLocalDatesTest {
     val localDate = LocalDate.of(2010, 10, 18)
     assertThat(localDate.localizedString).isEqualTo("18/10/10")
   }
+
+  @Test
+  fun getDateSeparator_US() {
```

Comment:
```suggestion
  fun `parse separator for US locale date pattern`
```
please also update the change at other places